### DISCUSSION TITLE
播放音频时控制界面不会自动消隐

### DISCUSSION THREAD

**Discussion by echoIIImk2** [hooke007/mpv_PlayKit#382] [mpv 通用]
播放纯音频文件，比如mp3时，控制界面会一直驻留着，不知道有没有办法像播放视频一样自动隐藏。

**Comment by hooke007** [comment]
It's the feature. 自行检查脚本选项

**Reply by hooke007** [reply]
另外为什么发英文区？
已转通用区

**Comment by echoIIImk2** [comment]
我看了这个不是mpv.conf控制的，似乎是scripts\uosc那一大堆脚本控制的，我觉得找到这个feature并进行修改已超过普通用户的能力，而希望播放音频文件与播放视频文件有同样的UI表现也不算过分的要求，没准就有热心人给出解决方案呢，希望对于删帖提出申诉。

Karl Hook ***@***.***> 于2023年11月5日周日 13:32写道：

**Reply by hooke007** [reply]
没人删你的帖
删帖了你还怎么回复？

**Reply by hooke007** [reply]
如果花一点时间看几行中文也是“已超过普通用户的能力”的话，那这个庞大的项目又是怎么来的呢？

https://github.com/hooke007/MPV_lazy/blob/d3ee1b069c170cdfd75085f68b06c9c2d9971981/portable_config/script-opts/uosc.conf#L16-L17
